### ISSUE TITLE
Babel Plugin Detected Under a Wrong Name and Cannot Be Ignored

### ISSUE THREAD

**Issue by portrik** [webpro/knip#363]
## Description

I ran into an issue with the detection of unused dependencies. I am in the process of adding knip to a larger project that uses Babel. However, knip recognizes a Babel plugin under a wrong name and reports it as an unlisted dependency. But when I add the recognized name to `ignoreDependencies` in `knip.jsonc`, it still gets reported as unlisted and the key in the configuration is reported as unused.

Curiously, `@kiwicom/babel-preset` goes through a similar transformation to `babel-plugin-@kiwicom` but it can be ignored with `ignoreDependencies`.

Currently, my workaround is to disable `unlisted`. Which is not ideal, obviously. Is this a knip, Babel or `@kiwicom/babel-*` issue?

## Minimal Reproduction
```javascript
// babel.config.js
module.exports = (api) => {
    return {
        presets: ['@kiwicom'],
        plugins: ['@kiwicom/orbit-components']
    };
};
```

```jsonc
// knip.jsonc
{
    "project": ["."],
    "ignoreDependencies": [
        "babel-plugin-@kiwicom/orbit-components",
        "babel-plugin-@kiwicom"
    ]
}
```

```jsonc
// package.json
{
  "devDependencies": {
    "@babel/cli": "^7.23.4",
    "@babel/core": "^7.23.3",
    "@babel/preset-env": "^7.23.3",
    "@kiwicom/babel-plugin-orbit-components": "^4.2.10",
    "@kiwicom/babel-preset": "^0.4.3",
    "knip": "^2.42.0"
  }
}
```

Running knip after installing dependencies gives me the following output:
```
Unused devDependencies (5)
@babel/cli                              package.json
@babel/core                             package.json
@babel/preset-env                       package.json
@kiwicom/babel-plugin-orbit-components  package.json
@kiwicom/babel-preset                   package.json
Unlisted dependencies (1)
babel-preset-@kiwicom  babel.config.js
Configuration issues (1)
Unused item in ignoreDependencies: babel-plugin-@kiwicom/orbit-components
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.43.0. See [Release 2.43.0](https://github.com/webpro/knip/releases/tag/2.43.0) for release notes._

**Comment by webpro** [comment]
The Babel plugin was a bit neglected for some time, hopefully things are improved now! Thanks for the report.

**Comment by portrik** [comment]
Wow, thanks for such a fast response! And especially for the fix

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.0.0-canary.4. See [Release 3.0.0-canary.4](https://github.com/webpro/knip/releases/tag/3.0.0-canary.4) for release notes._
